packages/components/input-number/src/input-number.vue

```diff
@@ -284,7 +284,7 @@ watch(
   (value) => {
     const userInput = verifyValue(data.userInput)
     const newValue = verifyValue(value, true)
-    if (!isNumber(userInput) && (!userInput || userInput !== newValue)) {
+    if (isNumber(userInput) === false || userInput !== newValue) {
```

Comment:
```suggestion
    if (!isNumber(userInput) || userInput !== newValue) {
```

Reply:
I think this is better to read the code, don't know what you think?

Reply:
`isNumber` will only return `true` or `false`. I think it is more concise to use ` logical NOT` directly.